docs/color-rules.md

```diff
@@ -48,7 +48,7 @@ Crosses
 Examples
 --------
 
-The following annotated examples apply the rules of the _twists_ table into the outlined shapes of the _crosses_ table.
+The following examples combine the color rules relating to _twists_ with the shapes representing the _crosses_.
```

Comment:
Not sure this expresses enough it combines the colors expressing the twists within a stitch and the cross marks expressing the twists between stitches.